ballerina/modules/bulk/tests/bulk_csv_delete.bal

```diff
@@ -18,7 +18,10 @@ import ballerina/log;
 import ballerina/test;
 import ballerina/lang.runtime;
 
-@test:AfterSuite {}
+@test:Config {
+    enable: true,
+    dependsOn: [queryXml]
```

Comment:
no need to explicitly set `enable` as true. Its true by default :) lets update similar places
```suggestion
    dependsOn: [queryXml]
```

Reply:
3adc801afa9ac122325288623fdf0ce52ebca47b